index.html

```diff
@@ -11,25 +11,24 @@
   <title>Portfolio</title>
 </head>
 <body>
-  <div class="status-bar">
-    <div class="hour">9:41</div>
-    <div class="wifi"></div>
-    <div class="signal"></div>
-    <div class="battery"></div>
-  </div>
-  <nav class="navbar">
-     
-    <div class="nav-name">Jhon Doe</div>
-    <div class="nav-menu"></div>      
-        
+  <nav class="navbar">     
+    <h1 class="nav-name"><a href="#">Jhon Doe</a></h1>
+    <div class="nav-space"></div>
+    <div class="nav-menu"><img src="./Images/menu.png" height="30px"></div>        
   </nav>
-  <div class="container">
+  <section class="container">
     <div class="primary-text">Hey There. I’m Alex</div>
     <div class="supporting-text">I’am a Software Developer</div>
     <div class="description">I can help you build a product , feature or website. 
       Look through some of my work and experience! If you like what you see and have a project you need coded, 
       don’t hesitate to contact me.</div>
```

Comment:
***[OPTIONAL]*** Kindly use specific tags for specific sections.

Such as: 
- For the title `Hey There. I`m Alex`  you could use `h1`.
- For `I'm a Software Developer` you could use `h2` or `h3` tags.
- For a paragraph, it's better to use a `p` tag inside the `div` tag.

Here is a [reference](https://stackoverflow.com/questions/28529390/ok-to-have-text-in-div-without-paragraph-tag) from Stackoverflow about why it's better to use a specific tag for a specific section. You can consider checking it.
`Though it is also an important issue, Kindly fix it before merging.`